dashboard/app/assets/javascripts/soundEffects.js

```diff
@@ -44,6 +44,30 @@ Sounds.prototype.play = function (soundId, options) {
   sound.play(options);
 };
 
+Sounds.prototype.playURL = function (url, options) {
```

Comment:
maybe s/`options`/`playbackOptions` just to disambiguate from Sound config objects